### ISSUE TITLE
Game crashes when opening the map editor with TS maps

### ISSUE THREAD

**Issue by Mailaender** [OpenRA/OpenRA#14548]
RA is not effected. Seems to be independent from the map chosen.

```
matthias@mTower:~/Entwicklung/OpenRA> mono --debug OpenRA.Game.exe Game.Mod=ts
Platform is Linux
Engine version is {DEV_VERSION}
Using SDL 2 with OpenGL renderer
Desktop resolution: 1920x1080
No custom resolution provided, using desktop resolution
Using resolution: 1920x1080
Using window scale 1,00
OpenGL version: 4.5.0 NVIDIA 384.98
Using default sound device
Internal mods:
        all: All mods ({DEV_VERSION})
        cnc: Tiberian Dawn ({DEV_VERSION})
        d2k: Dune 2000 ({DEV_VERSION})
        modcontent: Mod Content Manager ({DEV_VERSION})
        ra: Red Alert ({DEV_VERSION})
        ts: Tiberian Sun ({DEV_VERSION})
External mods:
        cnc-release-20171014: Tiberian Dawn (release-20171014)
        d2k-release-20171014: Dune 2000 (release-20171014)
        ra-release-20171014: Red Alert (release-20171014)
        ra-{DEV_VERSION}: Red Alert ({DEV_VERSION})
        ra2-{DEV_VERSION}: Red Alert 2 ({DEV_VERSION})
        ts-{DEV_VERSION}: Tiberian Sun ({DEV_VERSION})
Loading mod: ts
Exception of type `System.Collections.Generic.KeyNotFoundException`: The given key was not present in the dictionary.
  at System.ThrowHelper.ThrowKeyNotFoundException () [0x00000] in /home/abuild/rpmbuild/BUILD/mono-4.6.1/mcs/class/referencesource/mscorlib/system/throwhelper.cs:68 
  at System.Collections.Generic.Dictionary`2[TKey,TValue].get_Item (TKey key) [0x00021] in <dca3b561b8ad4f9fb10141d81b39ff45>:0 
  at OpenRA.Graphics.ModelRenderer.EnableFrameBuffer (OpenRA.Graphics.Sheet s) [0x00008] in /home/matthias/Entwicklung/OpenRA/OpenRA.Game/Graphics/ModelRenderer.cs:289 
  at OpenRA.Graphics.ModelRenderer.EndFrame () [0x00061] in /home/matthias/Entwicklung/OpenRA/OpenRA.Game/Graphics/ModelRenderer.cs:322 
  at OpenRA.Graphics.WorldRenderer.GenerateRenderables () [0x00152] in /home/matthias/Entwicklung/OpenRA/OpenRA.Game/Graphics/WorldRenderer.cs:124 
  at OpenRA.Graphics.WorldRenderer.Draw () [0x00091] in /home/matthias/Entwicklung/OpenRA/OpenRA.Game/Graphics/WorldRenderer.cs:143 
  at OpenRA.Game.RenderTick () [0x0006b] in /home/matthias/Entwicklung/OpenRA/OpenRA.Game/Game.cs:656 
  at OpenRA.Game.Loop () [0x00142] in /home/matthias/Entwicklung/OpenRA/OpenRA.Game/Game.cs:787 
  at OpenRA.Game.Run () [0x00042] in /home/matthias/Entwicklung/OpenRA/OpenRA.Game/Game.cs:805 
  at OpenRA.Game.InitializeAndRun (System.String[] args) [0x00011] in /home/matthias/Entwicklung/OpenRA/OpenRA.Game/Game.cs:253 
  at OpenRA.Program.Main (System.String[] args) [0x00050] in /home/matthias/Entwicklung/OpenRA/OpenRA.Game/Support/Program.cs:37
```

On @openSUSE with https://en.opensuse.org/SDB:NVIDIA_drivers installed.

**Comment by pchote** [comment]
Probably a regression from #14439.

**Comment by RoosterDragon** [comment]
Fixed by #14564.
